ide/provisioning/install-debian-packages.sh

```diff
@@ -68,10 +68,6 @@ echo Installing dependencies for the Android target, not including SDK / NDK...
 apt-get install $APTOPTS openjdk-8-jdk-headless vorbis-tools adb libtool
 echo
 
-echo Installing dependencies for KOBO
-apt-get --assume-yes install libav-tools
```

Comment:
This line was added in this very PR, and now remove it again? Why add it in the first place?

Reply:
> This commit incorporates all changes to install-debian-packages.sh
> available in the current master branch (bb41583) of
> xcsoar-docker-build [1].

As explained in the commit message c019eecc, I based these patches on lordfolken/xcsoar-docker-build@bb41583. Thus I first incorporated all changes in this repository and then added my patches.

As this clutters the xcsoar git history (and is harder to review, sry) I edited my patchset accordingly now.